test/Discourse/Middleware/DiscourseHandlerTest.php

```diff
@@ -49,10 +56,13 @@ public function testDispatchesDiscoursePostAndReturnsEmpty202Response(): void
 
         $dispatcher = $this->prophesize(EventDispatcherInterface::class);
         $dispatcher
-            ->dispatch(Argument::that(function ($post) {
+            ->dispatch(Argument::that(function ($post) use ($id) {
                 TestCase::assertInstanceOf(DiscoursePost::class, $post);
                 TestCase::assertSame('#qanda', $post->getChannel());
-                TestCase::assertSame('https://discourse.laminas.dev/t/some-topic/42/4242', $post->getPostUrl());
+                TestCase::assertSame(
+                    sprintf('https://discourse.laminas.dev/t/some-topic/42/%d', $id ?: 1),
```

Comment:
`$id` here can be `null` or `1` according to data provider.
It is not immediately obvious that null implies 1 per the current discord behavior. Add comment please.

Reply:
Good point; pushed a commit that adds a comment here.